src/components/totals/GoalProgress/index.js

```diff
@@ -74,13 +78,25 @@ module.exports = React.createClass({
 
   renderText: function() {
     var props = this.props;
+    var message;
+
+    if(!props.goal && props.fallbackToFundsRaised) {
+
```

Comment:
Can you leave out the empty lines at beginning and end of if/else blocks please?